### ISSUE TITLE
BootstrapVue does not work with Vite

### ISSUE THREAD

**Issue by pedzed** [bootstrap-vue/bootstrap-vue#5421]
### Reproducing

1. Create Vite app
    ```bash
    npx create-vite-app bootstrapvue-test
    cd bootstrapvue-test/
    ```
1. Test Vite app
    ```bash
    npm run dev
    ```
1. Optionally close server
1. Install BootstrapVue 
    ```bash
    npm install bootstrap-vue bootstrap
    ```

### Expected behavior

Application works just like before.

### Versions
```json
"dependencies": {
    "bootstrap": "^4.5.0",
    "bootstrap-vue": "^2.14.0",
    "vue": "^3.0.0-beta.14"
},
"devDependencies": {
    "vite": "^0.16.6",
    "@vue/compiler-sfc": "^3.0.0-beta.14"
}
```

**Environment:**
- OS: Ubuntu 18.04

**Comment by tmorehouse** [comment]
Note, BootstrapVue is currently not compatible with Vue 3 beta, due to breaking changes within Vue 3 (such as the non-existence of `Vue.extend(...)`).

**Comment by Remigius2011** [comment]
Hi,

Di you already have an idea on how to migrate bootstrap-vue to vue 3? Maybe it would make sense to create an issue to track progress.

**Comment by Remigius2011** [comment]
I have discovered Vue 3 compatibility on the roadmap: https://github.com/bootstrap-vue/bootstrap-vue/projects/9#card-27772756

**Comment by jackmu95** [comment]
Closed in favor of #5196.
